### ISSUE TITLE
Print --solver argument/variable in pytest headers

### ISSUE THREAD

**Issue by snorfalorpagus** [pywr/pywr#119]
Currently AppVeyor runs tests using glpk and lpsolve in the same instance. It's difficult to work out which is which. Add something like "solver: glpk" to the headers (conftest.py?)

```
===== testing package: pywr-0.1-np110py27_0 =====
running run_test.py
============================= test session starts ============================= 
platform win32 -- Python 2.7.11, pytest-2.8.5, py-1.4.31, pluggy-0.3.1
rootdir: C:\Miniconda-x64\conda-bld\work, inifile: 
collected 94 items 
 
..\work\tests\test_analytical.py .......x... 
..\work\tests\test_control_curves.py .. 
..\work\tests\test_core.py .....x........ 
..\work\tests\test_license.py ...... 
..\work\tests\test_parameters.py ....... 
..\work\tests\test_piecewise.py .. 
..\work\tests\test_recorders.py .......
..\work\tests\test_river.py ...
..\work\tests\test_run.py ....xx.xxx...xxx..x.x... 
..\work\tests\test_scenarios.py ..... 
..\work\tests\test_xml.py xxxxxxxxxxxxx 
 
=================== 69 passed, 25 xfailed in 11.02 seconds ==================== 
============================= test session starts ============================= 
platform win32 -- Python 2.7.11, pytest-2.8.5, py-1.4.31, pluggy-0.3.1 
rootdir: C:\Miniconda-x64\conda-bld\work, inifile:  
collected 94 items 
 
..\work\tests\test_analytical.py .......x... 
..\work\tests\test_control_curves.py .. 
..\work\tests\test_core.py .....x........ 
..\work\tests\test_license.py ...... 
..\work\tests\test_parameters.py ....... 
..\work\tests\test_piecewise.py .. 
..\work\tests\test_recorders.py ....... 
..\work\tests\test_river.py ... 
..\work\tests\test_run.py ....xx.xxx...xxx..x.x... 
..\work\tests\test_scenarios.py ..... 
..\work\tests\test_xml.py xxxxxxxxxxxxx 
 
==================== 69 passed, 25 xfailed in 6.73 seconds ==================== 
===== pywr-0.1-np110py27_0 OK =====
```

**Comment by snorfalorpagus** [comment]
Example `conftest.py` from another project:

```
import pytest

import fiona
import rasterio
import shapely


def pytest_report_header(config):
    headers = []
    headers.append('fiona version: {}'.format(fiona.__version__))
    headers.append('rasterio version: {}'.format(rasterio.__version__))
    headers.append('shapely version: {}'.format(shapely.__version__))
    return '\n'.join(headers)
```

**Comment by snorfalorpagus** [comment]
Implemented by e2ef954d8074760d67e66b442156d51e2b446a16.
